provider/consumer.py

```diff
@@ -24,6 +25,8 @@
 from datetime import datetime
 from threading import Thread, Lock
 
+local_dev = os.getenv('LOCAL_DEV', 'False')
```

Comment:
Not sure that you need to check this variable twice. Just pass in the check_ssl variable during consumer creation?